Error: the trait `Clone` is not implemented for `CookieIdentityPolicy`

## Expected Behavior
Code as shown in the example in [actix-identity/src/lib.rs](https://github.com/actix/actix-extras/blob/master/actix-identity/src/lib.rs) should work. Here is a complete `main.rs` file that should compile according to the actix-identity documentation but doesn't:
```
use actix_identity::{CookieIdentityPolicy, IdentityService};
use actix_web::{App, HttpServer};

#[actix_web::main]
async fn main() -> std::io::Result<()> {
    let identity_policy = CookieIdentityPolicy::new(&[0; 32])
        .secure(false);

    HttpServer::new(move || {
        App::new()
            .wrap(IdentityService::new(identity_policy))
    })
        .bind(("0.0.0.0", 8080u16))?
        .run()
        .await
}
```

The project's `Cargo.toml` file:
```
[package]
name = "test-actix-id"
version = "0.1.0"
edition = "2018"

[dependencies]
actix-identity = "0.4.0-beta.2"
actix-web = "4.0.0-beta.8"
```

## Current Behavior
```
error[E0277]: the trait bound `CookieIdentityPolicy: Clone` is not satisfied in `[closure@src/main.rs:9:21: 12:6]`
  --> src/main.rs:9:5
   |
9  |       HttpServer::new(move || {
   |  _____^^^^^^^^^^^^^^^_-
   | |     |
   | |     within `[closure@src/main.rs:9:21: 12:6]`, the trait `Clone` is not implemented for `CookieIdentityPolicy`
10 | |         App::new()
11 | |             .wrap(IdentityService::new(identity_policy))
12 | |     })
   | |_____- within this `[closure@src/main.rs:9:21: 12:6]`
   |
   = note: required because it appears within the type `[closure@src/main.rs:9:21: 12:6]`
   = note: required by `HttpServer::<F, I, S, B>::new`

error[E0277]: `Rc<actix_identity::cookie::CookieIdentityInner>` cannot be sent between threads safely
  --> src/main.rs:9:5
   |
9  |       HttpServer::new(move || {
   |  _____^^^^^^^^^^^^^^^_-
   | |     |
   | |     `Rc<actix_identity::cookie::CookieIdentityInner>` cannot be sent between threads safely
10 | |         App::new()
11 | |             .wrap(IdentityService::new(identity_policy))
12 | |     })
   | |_____- within this `[closure@src/main.rs:9:21: 12:6]`
   |
   = help: within `[closure@src/main.rs:9:21: 12:6]`, the trait `Send` is not implemented for `Rc<actix_identity::cookie::CookieIdentityInner>`
   = note: required because it appears within the type `CookieIdentityPolicy`
   = note: required because it appears within the type `[closure@src/main.rs:9:21: 12:6]`
   = note: required by `HttpServer::<F, I, S, B>::new`

error[E0277]: the trait bound `CookieIdentityPolicy: Clone` is not satisfied in `[closure@src/main.rs:9:21: 12:6]`
  --> src/main.rs:4:1
   |
4  |   #[actix_web::main]
   |   ^^^^^^^^^^^^^^^^^^ within `[closure@src/main.rs:9:21: 12:6]`, the trait `Clone` is not implemented for `CookieIdentityPolicy`
...
9  |       HttpServer::new(move || {
   |  _____________________-
10 | |         App::new()
11 | |             .wrap(IdentityService::new(identity_policy))
12 | |     })
   | |_____- within this `[closure@src/main.rs:9:21: 12:6]`
   | 
  ::: /Users/me/.cargo/registry/src/github.com-1ecc6299db9ec823/actix-web-4.0.0-beta.8/src/server.rs:55:27
   |
55 |       F: Fn() -> I + Send + Clone + 'static,
   |                             ----- required by this bound in `HttpServer`
   |
   = note: required because it appears within the type `[closure@src/main.rs:9:21: 12:6]`
   = note: this error originates in the attribute macro `actix_web::rt::main` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0277]: `Rc<actix_identity::cookie::CookieIdentityInner>` cannot be sent between threads safely
  --> src/main.rs:4:1
   |
4  |   #[actix_web::main]
   |   ^^^^^^^^^^^^^^^^^^ `Rc<actix_identity::cookie::CookieIdentityInner>` cannot be sent between threads safely
...
9  |       HttpServer::new(move || {
   |  _____________________-
10 | |         App::new()
11 | |             .wrap(IdentityService::new(identity_policy))
12 | |     })
   | |_____- within this `[closure@src/main.rs:9:21: 12:6]`
   | 
  ::: /Users/me/.cargo/registry/src/github.com-1ecc6299db9ec823/actix-web-4.0.0-beta.8/src/server.rs:55:20
   |
55 |       F: Fn() -> I + Send + Clone + 'static,
   |                      ---- required by this bound in `HttpServer`
   |
   = help: within `[closure@src/main.rs:9:21: 12:6]`, the trait `Send` is not implemented for `Rc<actix_identity::cookie::CookieIdentityInner>`
   = note: required because it appears within the type `CookieIdentityPolicy`
   = note: required because it appears within the type `[closure@src/main.rs:9:21: 12:6]`
   = note: this error originates in the attribute macro `actix_web::rt::main` (in Nightly builds, run with -Z macro-backtrace for more info)

error: aborting due to 4 previous errors
```

## Possible Solution
`CookieIdentityPolicy` should implement `Clone`.

## Steps to Reproduce (for bugs)
1. Create a new project with the Cargo.toml and main.rs files provided above.
2. Do `cargo run`.

## Your Environment

* Rust Version (I.e, output of `rustc -V`): rustc 1.55.0-nightly (71b8742bb 2021-07-03)
* Actix-* crate(s) Version: actix-identity = "0.4.0-beta.2"; actix-web = "4.0.0-beta.8"

--- a/actix-identity/src/lib.rs
+++ b/actix-identity/src/lib.rs
@@ -33,15 +33,23 @@
 //!     HttpResponse::Ok().finish()
 //! }
 //!
-//! // create cookie identity backend
-//! let policy = CookieIdentityPolicy::new(&[0; 32])
-//!     .name("auth-cookie")
-//!     .secure(false);
+//! #[actix_web::main]
+//! async fn main() -> std::io::Result<()> {
+//!     HttpServer::new(move || {
+//!         // create cookie identity backend
+//!         let policy = CookieIdentityPolicy::new(&[0; 32])
+//!             .name("auth-cookie");
+//!             .secure(false);
 //!
-//! let app = App::new()
-//!     // wrap policy into middleware identity middleware
-//!     .wrap(IdentityService::new(policy))
-//!     .service(services![index, login, logout]);
+//!         App::new()
+//!             // wrap policy into middleware identity middleware
+//!             .wrap(IdentityService::new(policy))
+//!             .service(services![index, login, logout])
+//!     })
+//!     .bind(("0.0.0.0", 8080u16))?
+//!     .run()
+//!     .await
+//! }
 //! ```
 
 #![deny(rust_2018_idioms, nonstandard_style)]

--- a/actix-identity/src/lib.rs
+++ b/actix-identity/src/lib.rs
@@ -7,7 +7,7 @@
 //!
 //! To access current request identity, use the [`Identity`] extractor.
 //!
-//! ```
+//! ```no_run
 //! use actix_web::*;
 //! use actix_identity::{Identity, CookieIdentityPolicy, IdentityService};
 //!

--- a/actix-identity/src/lib.rs
+++ b/actix-identity/src/lib.rs
@@ -38,7 +38,7 @@
 //!     HttpServer::new(move || {
 //!         // create cookie identity backend
 //!         let policy = CookieIdentityPolicy::new(&[0; 32])
-//!             .name("auth-cookie");
+//!             .name("auth-cookie")
 //!             .secure(false);
 //!
 //!         App::new()

